docs/pom.xml

```diff
@@ -40,11 +40,96 @@
                         </goals>
                         <configuration>
                             <target>
-                                <exec dir="${project.basedir}" executable="jekyll" failonerror="true">
+                                <available file="${project.build.directory}/che-repo" property="cheRepoExists" type="dir"/>
+                                <mkdir dir="${project.build.directory}/che-repo"/>
+                                <exec dir="${project.basedir}" executable="docker" failonerror="true">
+                                    <arg value="run" />
+                                    <arg value="--name" />
+                                    <arg value="jekyll_git_che" />
+                                    <arg value="jekyll/jekyll" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="cd /srv/jekyll/ ; git init ; git remote add -t add-docs-images -f origin https://github.com/eclipse/che.git ; git checkout add-docs-images" />
```

Comment:
Because it seems it's reinventing the wheel that something that is provided by maven
With maven, che repository could produce maven artifacts that could be used in codenvy ( or any other repository based on maven ( including artik, etc.) 
So, no need to parse sha-1, no need to clone repositories, no need to know the branch name, etc it's done by maven